changelog/new_add_internal_affairs_node_first_or_last_argument.md

```diff
@@ -0,0 +1 @@
+* [#12305](https://github.com/rubocop/rubocop/pull/12305): Require `rubocop-ast` version 1.30 or greater. ([@sambostock][])
```

Comment:
Can you rename `changelog/new_add_internal_affairs_node_first_or_last_argument.md` to `changelog/change_require_rubocop_ast_version_1_30_or_greater.md`?

Reply:
Done!